libproject/devstart.rb

```diff
@@ -37,6 +40,115 @@ def swagger_regen()
   :fn => Proc.new { |*args| swagger_regen(*args) }
 })
 
+def capitalize(str)
+   str.split('_').map {|x| x.capitalize}.join
+end
+
+ID_SUFFIX = "_id"
+
+def remove_id(id_column)
+  if not id_column.end_with? ID_SUFFIX then
```

Comment:
opt_nit: you could use `unless` rather than `if not` here. Maybe slightly more readable, but this is completely subjective since there are generally 8 different ways to do the same thing in Ruby, so feel free to ignore.

Reply:
Done.